activejob/lib/active_job/serializers/base_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module Serializers
+    class BaseSerializer
```

Comment:
We should add documentation for this class and here also put the example you put in the guides